src/test/java/com/gojek/beast/converter/ConsumerRecordConverterIntegrationTest.java

```diff
@@ -47,7 +52,15 @@ public void setUp() {
         columnMapping.put(2, "bq_order_url");
         columnMapping.put(3, "bq_order_details");
         rowMapper = new RowMapper(columnMapping);
-        recordConverter = new ConsumerRecordConverter(rowMapper, parser, clock);
+
+        System.setProperty("FAIL_ON_NULL_MESSAGE", "false");
+        AppConfig appConfig = ConfigFactory.create(AppConfig.class, System.getProperties());
+        recordConverter = new ConsumerRecordConverter(rowMapper, parser, clock, appConfig);
+
+        System.setProperty("FAIL_ON_NULL_MESSAGE", "true");
```

Comment:
instead of creating different recordConverters, lets just create one recordConverter in all the test functions. Right now the recordConverter is getting created in Before function, we can create it in each test function and add create AppConfig in each test, what do you think? I think we should avoid creating multiple recordConverters

Reply:
What's wrong with creating it multiple times? You need the flexibility to test different scenarios right?